CONTRIBUTING.md

```diff
@@ -13,7 +13,7 @@ You'll need a recent version of [nodejs](https://nodejs.org/en/) to work on RMWC
 > **NOTE**: If you expect to commit updated or new dependencies, please ensure you are using npm 5, which will
 > also update `package-lock.json` correctly when you install or upgrade packages.
 
-Once node is installed, simply clone our repo (or your fork of it) and run `npm install`
+Once node is installed, simply clone our repo (or your fork of it) and run `npm install`. You'll notice you have to install react and react-dom with the no-save option. These are peer dependencies that are not installed automatically.
```

Comment:
This does not apply anymore, right?

Reply:
React is still as a peer dependency in the package.json as I see it. So installing react is still needed right?

Reply:
Someone put in a PR that fixed it. Looked at other libraries, they still include React as a dev dependency which shouldn’t cause any problems when being used in a normal context with npm install.

Reply:
Ah, was looking at the v1.0.2 tag ... sorry.